lib/SlackBot.js

```diff
@@ -4,6 +4,7 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var querystring = require('querystring');
 var async = require('async');
+var Cryptr = require('cryptr');
```

Comment:
@tmundt the purpose of this is to securely store tokens in the backend store. Any compromise to the backed store should not leak tokens. The user can give a key in a configuration which can be used to encrypt tokens before storing. The same key can be used to decrypt the token. The decrypt function in same library like `Cryptr` can be used for that.